swap-serde/src/monero.rs

```diff
@@ -55,7 +56,7 @@ pub mod private_key {
         x.consensus_encode(&mut bytes)
             .map_err(|err| S::Error::custom(format!("{err:?}")))?;
         if s.is_human_readable() {
-            s.serialize_str(&hex::encode(bytes.into_inner()))
+            s.serialize_str(&data_encoding::HEXLOWER.encode(&bytes.into_inner()))
```

Comment:
did you make sure this matches the behaviour of the old `hex` crate exactly? If it doesn't, this will break network protocols.

Reply:
note to self: this should match because [hex::encode](https://docs.rs/hex/latest/hex/fn.encode.html) encodes as lower-case.

Reply:
Yeah, `hex` encodes as lcase and decodes as either; `data_encoding::HEX*_PERMISSIVE` decode as either and we only encode with `data_encoding::HEXLOWER`.